SConstruct

```diff
@@ -264,6 +264,16 @@ else:
 if env["precision"] == "double":
     env.Append(CPPDEFINES=["REAL_T_IS_DOUBLE"])
 
+# Disable exception handling. Godot doesn't use exceptions anywhere, and this
+# saves around 20% of binary size and very significant build time (GH-80513).
+if env["disable_exceptions"]:
+    if env.get("is_msvc", False):
+        env.Append(CPPDEFINES=[("_HAS_EXCEPTIONS", 0)])
+    else:
+        env.Append(CCFLAGS=["-fno-exceptions"])
```

Comment:
This isn't working in my testing, but if I change it to `CXXFLAGS` it does:
```suggestion
        env.Append(CXXFLAGS=["-fno-exceptions"])
```

Reply:
That's weird, `CCFLAGS` is equivalent to setting both `CFLAGS` and `CXXFLAGS`. So if this doesn't work in godot-cpp, that's a sign that we have a bigger issue.

Reply:
Well, exceptions _are_ a C++ specific thing, so it does sort of make sense that they should be in `CXXFLAGS` rather than `CFLAGS`. :-)

But, yeah, it certainly seems like there must be a larger issue somewhere, and that's worth figuring out.